instagram/src/components/CommentSection/CommentSection.js

```diff
@@ -1,13 +1,54 @@
 import React from 'react';
 import './CommentSection.css';
+import moment from 'moment';
 
 
 class CommentSection extends React.Component {
+   
+    constructor(props) {
+        super(props)
+        this.addNewComment = this.addNewComment.bind(this);
+        this.state = {
+            comments : this.props.comments.map(comment => comment.text),
+            username : this.props.comments.map(comment => comment.username)
+        }
+    }
 
-    render() {
+    addNewComment(e) {
```

Comment:
If you wanted to change this into an arrow function, it would look like this:
`addNewComment = e => {}`